src/config/rcxml.c

```diff
@@ -1010,10 +1015,19 @@ post_processing(void)
 		struct libinput_category *l = libinput_category_create();
 		assert(l && libinput_category_get_default() == l);
 	}
-	if (!wl_list_length(&rc.workspace_config.workspaces)) {
-		struct workspace *workspace = znew(*workspace);
-		workspace->name = xstrdup("Default");
-		wl_list_append(&rc.workspace_config.workspaces, &workspace->link);
+
+	int workspaces_configured = wl_list_length(&rc.workspace_config.workspaces);
+
+	if (workspaces_configured < rc.workspace_config.count) {
+		struct workspace *workspace;
+		char workspace_name[32]; // Maximum length of workspace name "Workspace X"
+
+		for (int i = workspaces_configured; i < rc.workspace_config.count; i++) {
+			workspace = znew(*workspace);
+			snprintf(workspace_name, sizeof(workspace_name), "Workspace %d", i);
```

Comment:
The `snprintf()` call misses a `+ 1` for the `i` value.
Currently it would create something like this with the example from above:
```
- 1st workspace
- 2nd workspace
- Workspace 2
- Workspace 3
```

The two empty lines (before `if()` and `for()`) can be removed.

Other than the missing `+ 1` and the empty line nitpick this seems to only miss the documentation update :)
Feel free to `Resolve` the comments that you took care of.